app/api/auth.py

```
@@ -1,3 +1,383 @@
+    Example protected route that requires authentication.
+    """
+    return {
+        "message": f"Hello {current_user.username}! This is a protected route.",
+        "user_id": current_user.id,
+        "timestamp": "2025-07-03T12:00:00Z"
+    }
+
+"""
+Authentication API routes.
+"""
+from datetime import timedelta
```

Ruff (0.11.9)

Redefinition of unused `timedelta` from line 4

Remove definition: `timedelta`

(F811)